crates/viewer/re_component_fallbacks/tests/all_component_fallbacks.rs

```diff
@@ -0,0 +1,55 @@
+use re_log_types::EntityPath;
+use re_test_context::TestContext;
+use re_types::{
+    ViewClassIdentifier,
+    external::arrow::util::display::{ArrayFormatter, FormatOptions},
+};
+use re_viewer_context::{BlueprintId, DataQueryResult, QueryContext, ViewContext};
+
+#[test]
```

Comment:
suprised you sticked it in here after all and not on `re_viewer` - I thought we wanted to have snapshots also for all views and have access to anything that's registered "late"?

Reply:
Currently views only have access to register view-specific fallbacks. So those wouldn't affect this test.

I think you mentioned doing the all views test as a follow-up? And I imagine that'd be separate from this